front/src/applications/operationalStudies/hooks/useScenarioData.ts

```diff
@@ -12,30 +12,39 @@ import {
 import useLazyProjectTrains from 'modules/simulationResult/components/SpaceTimeChart/useLazyProjectTrains';
 import { getOperationalStudiesElectricalProfileSetId } from 'reducers/osrdconf/operationalStudiesConf/selectors';
 import type {
+  PacedTrainResultWithPacedTrainId,
   TimetableItemId,
-  TrainId,
+  TimetableItemWithTimetableId,
   TrainScheduleId,
   TrainScheduleResultWithTrainId,
 } from 'reducers/osrdconf/types';
 import { getTrainIdUsedForProjection } from 'reducers/simulationResults/selectors';
+import { getShowPacedTrains } from 'reducers/user/userSelectors';
 import {
+  formatEditoastTrainIdToPacedTrainId,
   formatEditoastTrainIdToTrainScheduleId,
   formatTrainScheduleIdToEditoastTrainId,
 } from 'utils/trainId';
 import { mapBy } from 'utils/types';
 
 import useAutoUpdateProjection from './useAutoUpdateProjection';
+import useLazyLoadTimetableItems from './useLazyLoadTimatableItems';
 import useLazyLoadTrains from './useLazyLoadTrains';
 import usePathProjection from './usePathProjection';
 import formatTrainScheduleSummaries from '../helpers/formatTrainScheduleSummaries';
 
 const useScenarioData = (scenario: ScenarioResponse, infra: InfraWithState) => {
   const electricalProfileSetId = useSelector(getOperationalStudiesElectricalProfileSetId);
   const trainIdUsedForProjection = useSelector(getTrainIdUsedForProjection);
+  const showPacedTrains = useSelector(getShowPacedTrains);
 
+  // TODO Paced trains : remove these 3 states in https://github.com/OpenRailAssociation/osrd/issues/10791
   const [trainSchedules, setTrainSchedules] = useState<TrainScheduleResultWithTrainId[]>();
-  const [trainIdsToFetch, setTrainIdsToFetch] = useState<TrainId[]>();
-  const [trainIdsToProject, setTrainIdsToProject] = useState<Set<TrainId>>(new Set());
+  const [trainIdsToFetch, setTrainIdsToFetch] = useState<TimetableItemId[]>();
+  const [trainIdsToProject, setTrainIdsToProject] = useState<Set<TimetableItemId>>(new Set());
```

Comment:
You can remove trainSchedules and trainIdsToFetch and directly use timetableItems
```suggestion
  const [trainIdsToProject, setTrainIdsToProject] = useState<Set<TimetableItemId>>(new Set());
```

Reply:
Done. Made me change some parts of "not showPacedTrains" part but seems to bo still working :)